packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -192,17 +192,17 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
         { this._isDescriptionAvailable &&
           <span id={ this._descriptionId }>
             { description && <span className={ css('ms-TextField-description', styles.description) }>{ description }</span> }
+            <div aria-live='assertive'>
```

Comment:
shouldn't this move into the `errrorMessage` condition block? Otherwise we just add a div there with no content unnecessarily.

Also changes need a change file. Try running `npm run change` from the root to generate one.

Reply:
It was not working with NVDA old version. However, I updated to latest version `2018.1` and it is working fine. So, moved div into `errorMessage` condition block. Updated change file as well. Verified on firefox + NVDA and EDGE + narrator. Thanks.!